test/e2e-tests.sh

```diff
@@ -147,6 +147,15 @@ must_fail  "describe deleted resource" tkn resource describe skaffold-git
 must_fail  "show logs deleted taskrun" tkn taskrun logs test-template-volume
 must_fail  "describe deleted eventlistener" tkn eventlistener describe github-listener-interceptor
 
+# Hub tests
+echo "Hub"
+echo "....................."
+run_test "hub command exists" tkn hub --help
+run_test "hub search exists" tkn hub search --help
+run_test "hub get exists" tkn hub get --help
+must_fail "hub search cli" tkn hub --api-server=api.nonexistent.server search cli
+must_fail "hub get cli works" tkn hub --api-server=api.nonexistent.server get task cli
```

Comment:
Should we remove `works` as it is a failing one

Reply:
Agree that renaming without `works` would help avoid confusion.